linux/deb_run.sh

```diff
@@ -0,0 +1,38 @@
+# install Python3.8.2
+#https://linuxize.com/post/how-to-install-python-3-8-on-debian-10/
+sudo apt-get update -y && \
+sudo apt-get install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libsqlite3-dev libreadline-dev libffi-dev curl liblzma-dev libbz2-dev && \
+curl -O https://www.python.org/ftp/python/3.8.2/Python-3.8.2.tar.xz && \
+tar -xf Python-3.8.2.tar.xz && \
+cd Python-3.8.2 && \
+./configure --enable-optimizations && \
+make -j `nproc` && \
+sudo make altinstall && \
+python3.8 --version
+
+# git clone
+cd ~
+sudo apt-get install -y git
+git clone https://github.com/CoronaHakab/CoronaHakabModel.git
+cd ./CoronaHakabModel/
+
+# we'll work with docker branch
+git checkout docker
+
+# install the project dependencies
+sudo pip3.8 install --upgrade pip
+sudo pip3.8 install pipenv
+pipenv install --dev
```

Comment:
we can't expect client to install dev requirements

Reply:
OK.
but we need to think if if swimport is a dev package since we need for running if we don't deploy the binaries. I think building is the way to go instead of providing binaries as they might be different on different machines ? i.e. different arch or different cpu optimizations?